ui/src/timeMachine/components/TagSelector.scss

```diff
@@ -52,12 +52,20 @@ button.tag-selector--remove,
   pointer-events: none;
   flex: 1 1 0;
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
+  padding: 12px 12px 36px 12px;
```

Comment:
Can you use margin variables from `_variables.scss`?

Could probably do something like:
```
padding: $ix-marg-c;
padding-bottom: $ix-marg-d;
```

Reply:
Yep can do